From: mike.miller@hp.com <mike.miller@hp.com>
Date: Wed, 13 Dec 2006 12:08:56 +0000 (+0100)
Subject: [PATCH 1/2] cciss: map out more memory for config table
X-Git-Tag: v2.6.20-rc1
X-Git-Url: http://www.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commitdiff;h=98040015bc5d6cea3bd2dcb642fe1e8c4bded8e1

[PATCH 1/2] cciss: map out more memory for config table

This patch maps out more memory for our config table. It's required to reach
offset 0x214 to disable DMA on the P600. I'm not sure how I lost this hunk.
Please consider this for inclusion.

Signed-off-by: Mike Miller <mike.miller@hp.com>
Signed-off-by: Jens Axboe <jens.axboe@oracle.com>
---

--- a/drivers/block/cciss.c
+++ b/drivers/block/cciss.c
@@ -2865,7 +2865,7 @@ static int cciss_pci_init(ctlr_info_t *c
 #ifdef CCISS_DEBUG
 	printk("address 0 = %x\n", c->paddr);
 #endif				/* CCISS_DEBUG */
-	c->vaddr = remap_pci_mem(c->paddr, 200);
+	c->vaddr = remap_pci_mem(c->paddr, 0x250);
 
 	/* Wait for the board to become ready.  (PCI hotplug needs this.)
 	 * We poll for up to 120 secs, once per 100ms. */
